integration/build.gradle

```diff
@@ -84,6 +84,8 @@ dependencies {
     testCompile group: 'org.eclipse.jetty', name: 'jetty-server', version: versions.jetty
     testCompile group: 'org.eclipse.jetty', name: 'jetty-servlet', version: versions.jetty
     testCompile group: 'org.eclipse.jetty', name: 'jetty-servlets', version: versions.jetty
+    testCompile group: 'com.google.code.gson', name: 'gson', version: '2.8.5'
```

Comment:
Instead of adding this maybe we should update `javers-core` dependency to version with newer `gson`. What do you think about this?

Reply:
Yes, we can do that ;) I will add a commit for this.